scripts/apps/authoring/metadata/metadata.js

```diff
@@ -556,6 +556,33 @@ function MetaWordsListDirective() {
     };
 }
 
+
+MetaKeywordsListDirective.$inject = [];
+function MetaKeywordsListDirective() {
+    return {
+        scope: {
+            item: '=',
+            field: '@',
+            disabled: '=',
+            list: '=',
+            change: '&'
+        },
+        templateUrl: 'scripts/apps/authoring/metadata/views/metadata-keywords-list.html',
+        link: function(scope, element) {
+            scope.words = scope.list ? scope.list.join(' ') : '';
+
+            scope.changeList = function() {
+                var wordList = scope.words.split(' ');
+                var o = {};
+
+                o[scope.field] = wordList;
+                _.extend(scope.item, o);
```

Comment:
not sure why `o` is used here, why not just `scope.item[scope.field] = scope.words.split(' ' );` ?